seqr/views/apis/anvil_workspace_api.py

```diff
@@ -236,7 +248,95 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
         project_name=project.name,
         sample_type=sample_type,
         genome_version=GENOME_VERSION_LOOKUP.get(project.genome_version),
+        dag_name = "seqr_vcf_to_es_{anvil_type}_v{version}".format(anvil_type=sample_type, version=DAG_VERSION),
         dag=json.dumps(pipeline_dag, indent=4),
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+def _trigger_anvil_dags(project, data_path, user_input):
```

Comment:
this is very nitpicky, but we only will ever trigger one dag at a time so using the plural "dags" in this fuction name is confusing. I think also its better to use a more generic name explaining what is happening, something like `_trigger_data_loading` since DAG is not exactly a common-knowledge thing in coding. For the very dag-specific helper functions, like `_wait_for_dag_variable_update` it is okay to keep the word dag in the function name, but for this parent function I think its better to be less specific and more descriptive